src/Twig/ParameterExtension.php

```diff
@@ -4,14 +4,19 @@
 
 namespace StefanDoorn\SyliusGtmEnhancedEcommercePlugin\Twig;
 
+use StefanDoorn\SyliusGtmEnhancedEcommercePlugin\Object\ProductDetailInterface;
+use Sylius\Component\Core\Model\ProductInterface;
 use Twig\Extension\AbstractExtension;
 use Twig\TwigFunction;
 
 final class ParameterExtension extends AbstractExtension
```

Comment:
This one now breaks due to new parameters added. Can we add a second extension that registers `sylius_gtm_enhanced_ecommerce_product_identifier`? Will make upgrading easier for existing users.